test/color.py

```diff
@@ -0,0 +1,47 @@
+"""Test Color Sensor functionality"""
+import time
+import unittest
+
+from buildhat import ColorSensor
+
+
+class TestColor(unittest.TestCase):
+    """Test color sensor functions"""
+
+    def test_color(self):
+        """Test color sensor interval"""
+        color = ColorSensor('A')
+        color.avg_reads = 1
+        color.interval = 10
+        count = 1000
+        expecteddur = count * color.interval * 1e-3
```

Comment:
Might be more readable in snake_case

Reply:
You mean expeced_dur rather than expecteddur?

Reply:
Yeah